conda_forge_tick/auto_tick.xsh

```diff
@@ -184,9 +184,9 @@ def initialize_migrators():
     smithy_version = ![conda smithy --version].output.strip()
     pinning_version = json.loads(![conda list conda-forge-pinning --json].output.strip())[0]['version']
 
-    # add_rebuild($MIGRATORS, gx)
+    add_rebuild($MIGRATORS, gx)
```

Comment:
Please don't do this. It will restart the compiler builds, which are still pending some other fixes.